Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ruby:2.5.5-slim
+
+RUN apt-get update -qq && apt-get install -y build-essential libxml2-dev libxslt1-dev nodejs libpq-dev postgresql-client
+
+RUN gem install bundler -v 2.0.2
+
+RUN mkdir /traceo
+WORKDIR /traceo
```

Comment:
`WORKDIR` creates a directory if it doesn't exist, so no need to create it separately